deploy/crds/mattermost_v1alpha1_clusterinstallation_crd.yaml

```diff
@@ -79,6 +79,10 @@ spec:
             image:
               description: Image defines the ClusterInstallation Docker image.
               type: string
+            ingressAnnotations:
+              additionalProperties:
+                type: string
+              type: object
```

Comment:
Can you run `make yaml` to update the install files?

Reply:
done